server/Customer.java

```diff
@@ -65,12 +85,19 @@ public String getPassword() {
 		return password;
 	}
 
-	public void setAccountLocked(boolean value){
-		accountLocked = value;
-	}
-
 	public boolean isAccountLocked() {
 		return accountLocked;
 	}
 
+	public void setAccountLocked(boolean bool){
```

Comment:
You shouldn't call a parameter or a variable `bool`. For example, in C#, `bool` is the data type, so it's a really bad practice to do this. Same goes with `string`, `int` etc.